### ISSUE TITLE
scttreasury endpoints updates

### ISSUE THREAD

**Issue by freitasgouvea** [solid-world/solid-world-marketplace-backend#111]
Hey @cawabunga, @fmds07 and @lucasdcoutinh0, I created this task to list all changes and updates needed to be included in the backend.

**Comment by cawabunga** [comment]
1. Need to add `nft_name` and `nft_cover_image` to `/api/v1/scttreasury/offer`
2. Also need to add multiple offer cancelation to SCTCarbonTreasury contract.

<img width="218" alt="Screenshot 2022-04-28 at 09 25 05" src="https://user-images.githubusercontent.com/3716894/165751390-0e5dac7c-bb8c-46e4-9ef7-b75ca63acb8f.png">

_Project column (1.) and checkboxes (2.)_

**Comment by freitasgouvea** [comment]
@cawabunga I have included the sct treasury abi fix in this pull request https://github.com/solid-world/solid-world-dao-contracts/pull/23

**Comment by lucasdcoutinh0** [comment]
Add Treasury status to the projects 
![Uploading Screen Shot 2022-04-28 at 10.17.29.png…]()

**Comment by freitasgouvea** [comment]
Hey guys I created other task to batch accept offers https://github.com/solid-world/solid-world-marketplace-backend/issues/112

@lucasdcoutinh0 about the status of the projects I think we need to review how this feature can work, because I think the better way is store this information at nft table when we made some deposit using backoffice. Its not good idea call each project on sct treasury smart contract from backend at GET scttreasury/offer because this will consume much time to return the offer list. So first we have to update the nft model to contain this information and then update the scttreasury/deposit endpoint to update this field.
